### ISSUE TITLE
Revision History pane is not announced when opening

### ISSUE THREAD

**Issue by ebruchez** [orbeon/orbeon-forms#6630]

**Comment by ebruchez** [comment]
- Putting `role="dialog" aria-live="polite"` by hand on the element with `class="xbl-component xbl-fr-revision-history"` works.
- Curiously, it doesn't work if these attributes are placed on `fr-pane xforms-group`. Yet this is the element becoming visible.

**Comment by ebruchez** [comment]
- [ ] propagate `aria-live` attribute
- [ ] focus on content of pane when it is shown
